test/util/fake-xml-http-request-test.js

```diff
@@ -705,6 +705,44 @@ if (typeof window !== "undefined") {
             });
         });
 
+        describe(".setStatus", function () {
+            beforeEach(function () {
+                this.xhr = new FakeXMLHttpRequest();
+            });
+
+            it("cannot be set on closed request", function () {
+                assert.exception(function () {
+                    this.xhr.setStatus();
+                });
+            });
+
+            it("cannot be set on unsent request", function () {
+                this.xhr.open("GET", "/");
+
+                assert.exception(function () {
+                    this.xhr.setStatus();
+                });
+            });
+
+            it("by default sets status to 200", function () {
+                this.xhr.open("GET", "/");
+                this.xhr.send();
+                this.xhr.setStatus();
+
+                assert.equals(this.xhr.status, 200);
+                assert.equals(this.xhr.statusText, "OK");
+            });
+
+            it("sets status and status text", function () {
+                this.xhr.open("GET", "/");
+                this.xhr.send();
+                this.xhr.setStatus(206);
+
+                assert.equals(this.xhr.status, 206);
+                assert.equals(this.xhr.statusText, "Partial Content");
```

Comment:
Try to avoid using literal values for expected results, as they have no explanation of why they're expected or how we arrive at them.

A better approach would be to use variable names (or function calls) to show why this is the expected result.

The necessity is a bit exaggerated in this example, as the values are near-trivial. However, it's a good habit to build, to make code easier to understand. 

Consider questions like these:

* "why is this value expected to be `3`?"
* "is this `3` meant to be the same `3` as above, or is that an unhappy coincidence?"

Anyway, here's how I would restructure this test:

* split up the two expectations into two `it()` calls
* use variable names to explain how we arrive at the expected value

```javascript
it("sets status", function () {
    var expectedStatus = 206;
    var xhr = this.xhr;

    xhr.open("GET", "/");
    xhr.send();
    xhr.setStatus(expectedStatus);

    assert.equals(xhr.status, expectedStatus);
});

it("sets status text to the value from FakeXMLHttpRequest.statusCodes", function () {
    var status = 206;
    var expectedStatusText = FakeXMLHttpRequest.statusCodes[]
    var xhr = this.xhr;

    xhr.open("GET", "/");
    xhr.send();
    xhr.setStatus(status);

    assert.equals(xhr.statusText, expectedStatusText);
});
```